spring-cql/pom.xml

```diff
@@ -101,6 +101,12 @@
 			<artifactId>commons-lang3</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.1.7</version>
```

Comment:
Important: by default cassandra 3 brings in logback-core 1.1.3, which triggers an exception during the initialization of the loggers. Requesting 1.1.7 explicitly helps.